src/AidSelectionStep/AidSelectionStep.tsx

```diff
@@ -1,15 +1,49 @@
 import React, { FunctionComponent, useState } from 'react';
-import { Typography, Select, Alert } from 'antd';
+import { Typography, Select, Divider, Alert } from 'antd';
 import { InfoCircleOutlined } from '@ant-design/icons';
 
 import { NotebookNavAnchor, NotebookNavStep } from '../Notebook';
 import { TableSchema } from '../types';
 
+import { useNullAid } from './use-null-aid';
+
 import './AidSelectionStep.css';
 
 const { Title } = Typography;
 const { Option } = Select;
 
+type NullsInAidWarningProps = {
+  schema: TableSchema;
+  aidColumn: string;
+};
+
+const NullsInAidWarning: FunctionComponent<NullsInAidWarningProps> = ({ schema, aidColumn }) => {
+  const computedResult = useNullAid(schema, aidColumn);
+
+  switch (computedResult.state) {
+    case 'completed': {
+      return computedResult.value ? (
```

Comment:
The switch can be dropped and simplified to `return computedResult.state === 'completed' && computedResult.value ? ( ... ) : null`.